missions/2PzD_Template.VR/customization/loadouts/German/loadoutWHR39.sqf

```diff
@@ -460,11 +458,11 @@
         WHR39_Weapon_Secondary;
 
         //Primary Weapon
-        [Ger_Mag_MG_50_Mixed_sS,5] call Olsen_FW_FNC_AddItem;
         [Ger_Weap_MG34] call Olsen_FW_FNC_AddItem;
```

Comment:
Similar to how the later war German loadouts do this, we can replace this with the MG weapon macro that the Gruppe machine gunner also uses.